php-checks/src/main/java/org/sonar/php/checks/formatting/ControlStructureSpacingCheck.java

```diff
@@ -75,15 +74,15 @@ private void checkSpaceForStatement(FormattingStandardCheck formattingCheck, Ast
     for (AstNode semicolon : node.getChildren(PHPPunctuator.SEMICOLON)) {
       Token semicolonToken = semicolon.getToken();
       Token nextToken = semicolon.getNextAstNode().getToken();
-      int nbSpace = getNbSpaceBetween(semicolonToken, nextToken);
+//      int nbSpace = getNbSpaceBetween(semicolonToken, nextToken);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)